CODEOWNERS

```diff
@@ -26,6 +26,8 @@
 /Workbooks/Azure*Active*Directory*TroubleShooting/** @microsoft/azure-ad-workbooks @KranthiKiranerusu
 /Workbooks/Azure*Active*Directory/** @microsoft/azure-ad-workbooks @KranthiKiranerusu @sukhong
 
+/Workbooks/Azure*Active*Directory*Hybrid*Auth*Staged*Rollout/** @MichelleSwafford
```

Comment:
this would make you the *only* owner.  since you can't approve your own changes, and at least one owner would be required, you'd create a case where everyone *except* you can request a review but you're the only approver, so you yourself could never make changes to these files since an owner is required to approve.  if this is AAD related, that microsoft/azure-ad-workbooks "team" should be the owner for AAD related workbooks, so that your whole team can approve changes here